modules/rhizo_base/manifests/openbsc.pp

```diff
@@ -53,7 +53,8 @@
   }
 
   service { 'osmocom-nitb':
-      enable  => false,
+      ensure => 'running',
```

Comment:
We would need then also to modify the systemd control file with the right switches to use the MNCC socket.
currently, nitb MUST run from runit.

Reply:
I am not sure if this is the case with a fresh debian9 install. The osmocom-nitb packages I believe are installing a service file? I will double check today and confirm. When I was poking around it seemed like systemctl commands were the ones controlling the nitb, and after the install even before this change systemctl status was indicating to me that the nitb is running. There may be an interaction between runit and systemd here because the osmocom-nitb service is operating in legacy sysvinit support mode. I will investigate and clarify here.

Reply:
What's not the case? I assure you, osmo-nitb task will NOT use the MNCC socket out of the box. 
Yes, there is a mess between a sysvint task osmo-nitb and the osmocom-nitb service (or vice versa I don't really remember. 
Just use runit. It's easier.